service/history/ndc/history_replicator.go

```diff
@@ -64,12 +64,22 @@ type (
 		transactionManager transactionManager
 		logger             log.Logger
 
-		newBranchManager     branchManagerProvider
-		newConflictResolver  conflictResolverProvider
-		newWorkflowResetter  workflowResetterProvider
-		newStateBuilder      stateBuilderProvider
-		newMutableState      mutableStateProvider
-		newReplicationTaskFn newReplicationTaskFn
+		newBranchManager    branchManagerProvider
```

Comment:
Can you add `Fn` suffix to these function parameters?

Reply:
will do.